src/App.vue

```diff
@@ -412,36 +412,30 @@ export default class App extends Mixins(CommonMixin, FilingTemplateMixin) {
     // set current date from "real time" date from server
     this.setCurrentDate(DateUtilities.dateToYyyyMmDd(this.getCurrentJsDate))
 
-    // ensure user is authorized to access this business
+    // load account information
```

Comment:
There's a nice consistency to checking these things (account info, authorized actions, auth roles) together at the top of this function.